src/pages/LandingPage/LandingPage.jsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react'
```

Comment:
You can skip importing React in components other than root folder's index.js

Reply:
Thank you! Will keep that in mind.